src/python/yaramod_python.cpp

```diff
@@ -171,7 +171,7 @@ void addEnums(py::module& module)
 		.value("Comma", TokenType::COMMA)
 		.value("Private", TokenType::PRIVATE)
 		.value("Global", TokenType::GLOBAL)
-		.value("None", TokenType::NONE)
+		.value("NoneToken", TokenType::NONE)
```

Comment:
This should be safe because there is no real way to use the previous `None` name - it's a keyword, so `TokenType.None` returned error. I suppose `getattr(TokenType, "None")` would work, but is rather cumbersome. 

I intially wanted to keep both `None` and `NoneToken`, but Enums can't have duplicates.

Reply:
Correct, thanks for catching this bug! I also believe this should be safe.

Just a reference:
```
>>> import yaramod
>>> yaramod.TokenType.Global
<TokenType.Global: 51>
>>> yaramod.TokenType.None
  File "<stdin>", line 1
    yaramod.TokenType.None
                      ^^^^
SyntaxError: invalid syntax
```